### PR TITLE
Add instructions for Maven and manual Java users.

### PR DESCRIPTION
**Pull request by judy2k**
Been getting questions about installation with Maven and use without a dependency manager, so I've added some instructions, including general advice to use *something*.

## Contibution Checklist
* [x] My name is in [CONTRIBUTORS.md]

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,6 +19,8 @@ need a Nexmo account. Sign up [for free at nexmo.com][signup].
 
 To use the client library you'll need to have [created a Nexmo account][signup].
 
+### Groovy
```
Comment (leggetter) [inline README.md]: Gradle... I'm assuming

File: README.md [hunk 2]
```diff
@@ -33,13 +35,50 @@ dependencies {
 }
 ```
 
+### Maven
+
+Add the following to the correct places in your project's pom file:
```
Comment (leggetter) [inline README.md]: capitalise POM?

File: README.md [hunk 3]
```diff
@@ -33,13 +35,50 @@ dependencies {
...
+<dependency>
+      <groupId>com.nexmo</groupId>
+      <artifactId>client</artifactId>
+      <version>2.0.0-SNAPSHOT</version>
+</dependency>
+```
+
+### Build It Yourself
+
 Alternatively you can clone the repo and build the jar yourself:
```
Comment (leggetter) [inline README.md]: capitalise JAR or make it code?

File: README.md [hunk 4]
```diff
@@ -33,13 +35,50 @@ dependencies {
...
 git clone git@github.com:nexmo/nexmo-java.git
 gradle build
 ```
 
+### Download everything in a zip file
+
+**Note**: We *strongly recommend* that you use a tool that supports dependency management, such as [Gradle], [Maven],
+or [Ivy]
+
+We provide a [zip file](https://github.com/Nexmo/nexmo-java/releases/download/v2.0.0-SNAPSHOT/client-2.0.0-SNAPSHOT.zip)
```
Comment (leggetter) [inline README.md]: Do we want to hard code to 2.0.0-SNAPSHOT? Or create instructions to point to `releases` and go from there?

File: README.md [hunk 5]
```diff
@@ -33,13 +35,50 @@ dependencies {
...
+containing the client library jar, along with all the dependencies. Download the file, unzip it, and add the jar files
```
Comment (leggetter) [inline README.md]: JAR? see above

File: README.md (hunk 1 above)
Comment (judy2k) [inline README.md]: Ah yes - sometimes I even type "git" when I mean "gradle".

File: README.md (hunk 4 above)
Comment (judy2k) [inline README.md]: I like the direct link, but I suppose linking to Releases means I'm less likely to forget to update.

File: README.md [hunk 6]
```diff
@@ -33,13 +35,50 @@ dependencies {
...
 git clone git@github.com:nexmo/nexmo-java.git
 gradle build
 ```
 
+### Download everything in a ZIP file
+
+**Note**: We *strongly recommend* that you use a tool that supports dependency management, such as [Gradle], [Maven],
+or [Ivy]
+
+We provide a [ZIP file](https://github.com/Nexmo/nexmo-java/releases/)
```
Comment (leggetter) [inline README.md]: Each [release](#] contains a ZIP file...

?

Comment (judy2k) [inline README.md]: I changed it to "We release a [ZIP file for each release](link to releases page). Does that cover your intent?

**Pull request merged**